src/mixxx.cpp

```diff
@@ -1918,6 +1943,9 @@ void MixxxMainWindow::rebootMixxxView() {
     QPoint initPosition = pos();
     QSize initSize = size();
 
+    delete m_pCOShowSamplers;
```

Comment:
Add a comment why this is required. 
Something about that the skin expects to create a new CO and you reconnect to that after creation if so.